xml/System.Windows.Forms/PictureBox.xml

```diff
@@ -1192,7 +1192,36 @@
       <Parameters />
       <Docs>
         <summary>Displays the image specified by the <see cref="P:System.Windows.Forms.PictureBox.ImageLocation" /> property of the <see cref="T:System.Windows.Forms.PictureBox" />.</summary>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[## Remarks
+
+Starting with .NET 8, the behavior of loading a remote image by the `PictureBox` control changed. By default, the <xref:System.Net.ServicePointManager.CheckCertificateRevocationList%2A?displayProperty=fullName> property is set to `true` before a remote image is downloaded through <xref:System.Net.WebClient>. This setting ensures that servers with certificates have those certificates checked against the certificate authority revocation list (CRL) as part of the validation process.
+
+A previously working remote resource might fail to load when the the locally cached CRL is out-of-date, and an update can't be retrieved. This can happen when the network the app is running on is restricted and the CRL location isn't on the allowlist.
```

Comment:
```suggestion
A previously working remote resource might fail to load when the locally cached CRL is out-of-date and an update can't be retrieved. This can happen when the network the app is running on is restricted and the CRL location isn't on the allowlist.
```